rpms/pulp-puppet/pulp-puppet.spec

```diff
@@ -0,0 +1,245 @@
+%global required_pulp_version 2.8.2
+
+
+Name:		pulp-puppet
+Version:	2.8.2
+Release:	1%{?dist}
+
+License:	GPLv2
+Summary:	Support for Puppet content in the Pulp Platform
+URL:		https://github.com/pulp/pulp_puppet
+Source0:	https://github.com/pulp/pulp_puppet/archive/%{name}-%{version}-1.tar.gz
+BuildArch:	noarch
+
+BuildRequires:	python2-devel
+BuildRequires:	python2-setuptools
+BuildRequires:	python2-sphinx
+
+
+%description
+Provides a collection of platform plugins, client extensions and agent
+handlers that provide Puppet support
+
+
+%prep
+%autosetup -n pulp_puppet-%{name}-%{version}-1
+
+
+%build
+for directory in $(find . -type f -name "setup.py" | xargs dirname)
+do
+    pushd $directory
+    %py2_build
+    popd
+done
+
+pushd docs
+make %{?_smp_mflags} html
+# We don't want to install the objects.inv, because it is a build-time database
+rm _build/html/objects.inv
+popd
+
+
+%install
+for directory in $(find . -type f -name "setup.py" | xargs dirname)
+do
+    pushd $directory
+    %py2_install
+    popd
+done
+
+install -d %{buildroot}/%{_sysconfdir}/httpd/conf.d/
+install -d %{buildroot}/%{_sysconfdir}/pulp/admin/conf.d/
+install -d %{buildroot}/%{_sysconfdir}/pulp/agent/conf.d/
+install -d %{buildroot}/%{_sysconfdir}/pulp/vhosts80
+install -d %{buildroot}/%{_var}/lib/pulp/published/puppet/http
+install -d %{buildroot}/%{_var}/lib/pulp/published/puppet/https
+install -d %{buildroot}/%{_usr}/lib/pulp/agent/handlers
+install -d %{buildroot}/%{_usr}/lib/pulp/admin/extensions
+install -d %{buildroot}/%{_bindir}
+install -d %{buildroot}/%{_datadir}/pulp/wsgi/
+
+install -pm644 pulp_puppet_plugins/etc/httpd/conf.d/* \
+    %{buildroot}/%{_sysconfdir}/httpd/conf.d/
+
+install -pm644 pulp_puppet_plugins/etc/pulp/vhosts80/* \
+    %{buildroot}/%{_sysconfdir}/pulp/vhosts80
+
+install -pm644 pulp_puppet_plugins/usr/share/pulp/wsgi/* \
+    %{buildroot}/%{_datadir}/pulp/wsgi/
+
+install -pm644 pulp_puppet_extensions_admin/etc/pulp/admin/conf.d/* \
+    %{buildroot}/%{_sysconfdir}/pulp/admin/conf.d/
+
+install -pm644 pulp_puppet_handlers/etc/pulp/agent/conf.d/* \
+    %{buildroot}/%{_sysconfdir}/pulp/agent/conf.d/
+
+
+# ---- Admin Extensions
+%package admin-extensions
+Summary:	The Pulp Puppet admin client extensions
+Requires:	pulp-admin-client = %{required_pulp_version}
+Requires:	python2-pulp-puppet-common = %{version}
+Requires:	python2-setuptools
```

Comment:
@jeremycline I created https://github.com/pulp/packaging/pull/21 to remove the unneeded requires lines from the plugins. I also fixed this in Fedora.